environment_setup/requirements.txt

```diff
@@ -4,4 +4,5 @@ azureml>=0.2
 azureml-sdk>=1.0
 python-dotenv>=0.10.3
 flake8
-flake8_formatter_junit_xml
\ No newline at end of file
+flake8_formatter_junit_xml
+azure-cli==2.0.46
```

Comment:
This is an extremely old version of `az` - could we do a quick test with the latest 2.0.71?

Reply:
Done. Works with 2.0.71